calculate_error.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
```

Comment:
Komentarz do całości pliku -> formatowanie
Brak spacji po przecinkach, ale odstępy między funkcjami przepisowo 👍 

Jak nie chce ci się samej (zupełnie zrozumiałe ja np. nie formatuję sama plików) to:
w terminalu `pip install black` + dla notebooków `pip install jupyter-black jupyter`
I teraz przed każdym Commitem:
`black {path_to_file_to_format}`